functions/go/set-namespace/examples/expected/deploy.yaml

```diff
@@ -0,0 +1,40 @@
+#
```

Comment:
We should standardize the directory structure for the following:

- Example packages. Currently there's https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/examples
- E2E tests. Currently there's https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/tests